translation/source/mobile-misc.json

```diff
@@ -0,0 +1,17 @@
+{
+  "apiUnsupported": "Your version of lichess app is too old! Please upgrade for free to the latest version.",
+  "apiDeprecated": "Upgrade for free to the latest lichess app! Support for this version will be dropped on %s.",
+  "resourceNotFoundError": "Resource not found.",
+  "lichessIsUnavailableError": "lichess.org is temporarily down for maintenance.",
+  "lichessIsUnreachable": "lichess.org is unreachable.",
+  "mustSignInToJoin": "You must sign in to join it.",
+  "playerisInvitingYou": "%s is inviting you",
```

Comment:
not sure we should keep translations for challenges; maybe there is a way to use the website ones.

Reply:
The website communicates the challenge almost entirely through iconography:
![Screen Shot 2021-03-20 at 9 50 36 AM](https://user-images.githubusercontent.com/569991/111871695-eadb3680-8961-11eb-8858-ce4a5072b2bb.png)

We could easily get rid of the `toATypeGame` key by adopting the website's "Game Type · Time Control · Variant" string format - I can open a separate PR for that.

Re: the `playerisInvitingYou` key: I think the card would need to be reworked a little bit to get rid of it.
![Screen Shot 2021-03-20 at 9 50 58 AM](https://user-images.githubusercontent.com/569991/111871779-4c030a00-8962-11eb-9e03-068e85d87941.png)

Any suggestions?

Reply:
I liked how you tweaked it in the other PR. So we keep only the `playerIsInvitingYou` key